mocha.js

```diff
@@ -1461,7 +1461,7 @@ exports = module.exports = Mocha;
 if (typeof process !== 'undefined' && typeof process.cwd === 'function') {
   var join = path.join
     , cwd = process.cwd();
-  module.paths.push(cwd, join(cwd, 'node_modules'));
+  module && module.paths && module.paths.push(cwd, join(cwd, 'node_modules'));
```

Comment:
Todo: add documentation: 

// practicalmeteor change - make sure module.path is defined